tools/mosdepth.wdl

```diff
@@ -21,10 +21,11 @@ task coverage {
         coverage_bed: "BED file to pass to the `-b` flag of `mosdepth`. This will restrict coverage analysis to regions defined by the BED file."
```

Comment:
Do we think this would be `common`? I think I would lean towards "yes".

Reply:
Or are we assuming in the future it would surface due to being 'optional'?

Reply:
My assumption was that we would automatically surface all `optional` parameters. I kept my eye out for them while going through all these files, and I think it always makes sense to give them special treatment.
There aren't many places we use optional parameters. We typically default to an empty array or other such "null-esque" value.